sandbox/grommet-app/src/pages/layouts/kinds/OpsRamp/ResourceDetails.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Button,
+  Menu,
+  Text,
+  Layer,
+  Heading,
+  ResponsiveContext,
+} from 'grommet';
+import { LinkNext, StatusGoodSmall } from 'grommet-icons';
+import ContentPane from '../../../../components/ContentPane';
+import { ResourceAlerts } from './ResourceAlerts';
+import { ResourceTraits } from './ResourceTraits';
+import { ResourceOverview } from './ResourceOverview';
+
+interface ActionItem {
+  label: string | JSX.Element;
+  onClick: () => void;
+}
+
+const actionItems: ActionItem[] = [
+  { label: 'View details', onClick: () => {} },
+  { label: 'Copy ID', onClick: () => {} },
+  { label: 'Add relationships', onClick: () => {} },
+  { label: 'Assign credential', onClick: () => {} },
+  { label: 'Assign custom attribute', onClick: () => {} },
+  { label: 'Assign monitoring template', onClick: () => {} },
+  { label: 'Add maintenance window', onClick: () => {} },
+  { label: 'Add note', onClick: () => {} },
+  { label: 'Unmanage', onClick: () => {} },
+  {
+    label: <Text color="status-critical">Remove</Text>,
```

Comment:
Use text color instead

```suggestion
    label: <Text color="text-critical">Remove</Text>,
```